### DISCUSSION TITLE
Terminal Buffer

### DISCUSSION THREAD

**Discussion by iamevie** [helix-editor/helix#6831] [Ideas]
In my nvim config, I could open a terminal buffer in a split and use that as a terminal. This was really handy to for example to run Bacon or just any command. I'd like to do that in helix too!

Kind Regards :)

**Comment by David-Else** [comment]
There is no terminal in Helix at this time, you can get similar functionality using Kitty terminal windows, here is a script that uses them: https://github.com/David-Else/kitty-terminal-project-launcher
